viral/ensemble_reactivation.py

```
@@ -0,0 +1,585 @@
+HERE = Path(__file__).parent
+sys.path.append(str(HERE.parent))
+sys.path.append(str(HERE.parent.parent))
+
+from viral.constants import TIFF_UMBRELLA
+from viral.models import Cached2pSession, GrosmarkConfig, SortedPlaceCells
+from viral.rastermap_utils import (
+    load_data,
+    align_validate_data,
+    process_trials_data,
+    filter_speed_position,
+    filter_out_ITI,
```

Ruff (0.8.2)

`viral.rastermap_utils.filter_out_ITI` imported but unused

Remove unused import: `viral.rastermap_utils.filter_out_ITI`

(F401)